unearthed-arcana/2015/20151207.xml

```diff
@@ -136,4 +214,4 @@
 			<set name="isRitual">false</set>
 		</setters>
 	</element>
-</elements>
\ No newline at end of file
+</elements>
```

Comment:
also it seems you deleted line after `</elements>`. We prefer to keep empty line at the end of the file.

Reply:
oh wait, nevermind, you add the line, sorry, misread that